### ISSUE TITLE
Extracting image in predicted bounding box

### ISSUE THREAD

**Issue by saanasum** [experiencor/basic-yolo-keras#162]
Thanks for providing this great tool. 

I used your pipeline to predict different cell types in a FISH image. Works well! 

However, the classification probabilities for the predicted classes of the individual bounding boxes are not as high as it should be (between 0.35 and 0.75). 

Therefore, I was wondering whether it is possible to extract the small image (individual cell) in the predicted bounding boxes to be fed into a new CNN (more specialized in classifying the different cell types) for validating the classification to reach accuracies above 0.95 (which is of interest for medical purposes).

**Comment by experiencor** [comment]
@saanasum You may try to increase ```class_scale``` in config.json to penalize wrong class prediction more.

**Comment by experiencor** [comment]
@saanasum During training. The key is to make sure that the training set looks like the validation set and the test set. Removing the background in the test set is more difficult than adding random backgrounds to the training set.

**Comment by saanasum** [comment]
Although classification of detected objects is in principle working, unfortunately, the accuracies do not increase much. I do not get accuracies above 0.95. Are there further parameters that I can modify to potentially increase accuracy for each bounding box (cell).

**Comment by experiencor** [comment]
@saanasum I don't know of any other ways to improve classification accuracy. ```class_scale``` is the single parameter that controls the amount to penalize wrong class prediction. Why don't you try a very high value of this parameter (such as 100) to see how it affect the classification accuracy?

**Comment by saanasum** [comment]
I added the following in the utils.py:

```
def save_boxes(image, boxes, labels):    
    for box in boxes:
        xmin  = int((box.x - box.w/2) * image.shape[1])
        xmax  = int((box.x + box.w/2) * image.shape[1])
        ymin  = int((box.y - box.h/2) * image.shape[0])
        ymax  = int((box.y + box.h/2) * image.shape[0])	
	# write boxes as separate image file 
	# https://stackoverflow.com/questions/46879245/saving-contents-of-bounding-box-as-a-new-image-opencv-python
	name = 'nuclei' + labels[box.get_label()] + '_' + str(box.get_score()) + '_.jpg'
	foldername = '### to be defined ###'
	roi = image[ymin:ymax, xmin:xmax]
	cv2.imwrite(os.path.join(foldername, name), roi)        
    return image
```

and the following in the predict.py:

`image = save_boxes(image, boxes, config['model']['labels'])`
